CHANGELOG.md

```diff
@@ -14,6 +14,14 @@ Please add your changelog entry under this comment in the correct category (Secu
 ### Fixed
 * Fix deprecation warning for set-output in internal docker-scan job
 
+### Added
+* Register optional json log format
```

Comment:
nearly perfect - please link yourself and this PR here as well

Reply:
grr :D added in latest commit